chargebee/src/main/java/com/chargebee/android/CBResult.kt

```diff
@@ -69,29 +69,29 @@ internal fun <T> responseFromServer(response: Response<T?>): ChargebeeResult<T>
                     return ChargebeeResult.Error(
                         exp = CBException(
                             error = ErrorDetail(
-                                response.errorBody()?.string()
+                                response.errorBody()?.string(), httpStatusCode = response.code()
```

Comment:
How would the error handling change for the end customer?

Reply:
There is the issue raised by customer in flutter github and always error code being returned as null. based on response code they might have some logic in application level. The issue link updated in PR description.

Reply:
Thanks @cb-amutha. Might be useful if we update the docs corresponding to error handling as well.

Reply:
@cb-haripriyan Mainly we want to display "purchase was cancelled" separately from "purchase failed". We want to decide this based on the error code. Right now, we do string matching on the error message, but that's obviously not optimal. Also, an error code would help our error reporting since we can figure out which issues appear often.

Reply:
Thanks @ciriousjoker. It makes absolute sense. Reg. passing the store's error, let us look into the optimal way of handling it alongside the Chargebee codes.cc: @cb-amutha @cb-sriramthiagarajan